packages/flutter/lib/src/rendering/binding.dart

```diff
@@ -415,3 +415,20 @@ class RenderingFlutterBinding extends BindingBase with GestureBinding, ServicesB
     renderView.child = root;
   }
 }
+
+/// Client for listening to system fonts change.
+///
+/// Using this mixin will cause it automatically subscribe to system channel.
+mixin SystemFontsClient on RenderObject {
+  @override
+  void attach(covariant Object owner) {
+    super.attach(owner);
+    PaintingBinding.instance?.systemFonts?.addListener(markNeedsLayout);
```

Comment:
Why would instance or systemFonts be null? Shouldn't they always be initialized when you get to this place?